cmd/context/delete.go

```diff
@@ -28,12 +29,18 @@ import (
 func DeleteCMD() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "delete",
-		Args:  utils.ExactArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
-		Short: "Delete a context",
+		Args:  utils.MinimumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
+		Short: "Delete one or more contexts",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			args[0] = okteto.AddSchema(args[0])
-			args[0] = strings.TrimSuffix(args[0], "/")
-			return Delete(args[0])
+			for _, arg := range args {
+				arg = okteto.AddSchema(arg)
+				arg = strings.TrimSuffix(arg, "/")
+				if err := Delete(arg); err != nil {
```

Comment:
inside `Delete` function we are getting okteto context[ from storage](https://github.com/jeremyphua/okteto/blob/feature/support-deleting-multiple-context/cmd/context/delete.go#L50) for every context to delete. Instead of that we should send to `Delete` function all context to delete and iterate over contexts to delete them once we instantiate context storage.

Reply:
If deletion of one context fails because contexts doesn't exist it prevents deleting the rest of contexts so I propose to log an error but be able to delete the rest of contexts. WDYT about that UX @rlamana ?

Reply:
Hey! To clarify, if more than one context fails we return back all the errors?

Reply:
Agree with @AdrianPedriza, we should in that case delete the contexts that can be deleted but log an error for the one that has failed. If more than one error happens we will show them all, as long as we specify the name of the context that failed. As an example of output:
```
 ❌ 'xxxxx' doesn't exist and couldn't be deleted
 ❌ 'yyyyy' doesn't exist and couldn't be deleted
 ✅ 'zzzzz' deleted successfully
 ```

Reply:
Btw, the `usage` section should also be updated, it's not only flags that this command accepts
<img width="356" alt="Screenshot 2023-08-21 at 14 05 20" src="https://github.com/okteto/okteto/assets/237819/bda843a5-4050-4b06-8091-5890c7eb327f">